### PR TITLE
Add collection version content list command

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

File: pulpcore/cli/ansible/content.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+_ = gettext.gettext
+
+
+@click.group()
+@click.option(
+    "-t",
+    "--type",
+    "content_type",
+    type=click.Choice(["collection-version"], case_sensitive=False),
+    default="collection-version",
```
Comment (mdellweg) [inline pulpcore/cli/ansible/content.py:18]: I'm wondering if we should expose this as a "collection" to the user.

Comment (mdellweg) [reply pulpcore/cli/ansible/content.py:18]: Let's go for consistency.

Comment (fao89) [reply pulpcore/cli/ansible/content.py:18]: the closest we get to the galaxy user experience, the better

File: pulpcore/cli/ansible/content.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+)
+@pass_pulp_context
+@click.pass_context
+def content(ctx: click.Context, pulp_ctx: PulpContext, content_type: str) -> None:
+    if content_type == "collection-version":
+        ctx.obj = PulpAnsibleCollectionVersionContext(pulp_ctx)
+    else:
```
Comment (mdellweg) [inline pulpcore/cli/ansible/content.py:25]: Any chance we can add the code path for roles right now? Also OK to do it later.

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
The failures seem unrelated now. Have any of the containers been rebuilt?

**Pull request merged**